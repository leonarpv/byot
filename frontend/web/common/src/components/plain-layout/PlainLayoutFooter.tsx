import * as React from 'react';
import {Box, Typography} from '@material-ui/core';
import {useTranslation} from 'react-i18next';

interface Props {}

export const PlainLayoutFooter: React.FC<Props> = (props: Props) => {
  const {t} = useTranslation();
  return (
    <Box alignItems="center" justifyContent="center" display="flex">
      <Typography>{t('All rights reserved to byoT')}</Typography>
    </Box>
  );
};
