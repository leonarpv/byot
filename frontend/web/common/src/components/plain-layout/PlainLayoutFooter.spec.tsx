import * as React from 'react';
import {render} from '@testing-library/react';
import {PlainLayoutFooter} from './PlainLayoutFooter';

describe('<PlainLayoutFooter/>', () => {
  it('should render', () => {
    const {container} = render(<PlainLayoutFooter />);
    expect(container).toMatchSnapshot();
  });
});
