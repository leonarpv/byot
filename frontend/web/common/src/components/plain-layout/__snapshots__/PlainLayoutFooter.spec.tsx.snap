// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlainLayoutFooter/> should render 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-1"
  >
    <p
      class="MuiTypography-root makeStyles-typography-2 MuiTypography-body1"
    >
      All rights reserved to
      <strong>
         byoT
      </strong>
    </p>
  </div>
</div>
`;
